### DISCUSSION TITLE
add emoji-picker component for value mapping & more

### DISCUSSION THREAD

**Discussion by zuchka** [grafana/grafana#42697] [Dashboards, Panels & Visualizations]
You can use emoji in value mapping, which is awesome:

https://user-images.githubusercontent.com/37156449/144538195-b85bfe54-1fa0-40ce-b64d-82fcef1f866b.mp4

but you must copy/paste the emoji in the unicode browser format ([the browser column on Unicode's official chart](https://unicode.org/emoji/charts/full-emoji-list.html)).

An emoji picker component would make this task much easier. It would also signal to users in the UI that value mapping emoji is possible

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
